ml-agents/mlagents/trainers/trainer/trainer.py

```diff
@@ -121,7 +121,7 @@ def export_model(self, name_behavior_id: str) -> None:
         Exports the model
         """
         policy = self.get_policy(name_behavior_id)
-        settings = SerializationSettings(policy.model_path, policy.brain.brain_name)
```

Comment:
gross

Reply:
With PSankalp's change this stuff should go live in the policy anyways.